docsource/installation.rst

```diff
@@ -0,0 +1,208 @@
+Installation Instructions
+=========================
+
+Local Deployment
+-----------------
+Follow these instructions for a local deployment once the :ref:`System Requirements` have been satisfied. These instructions are known to work on Ubuntu 16.04 LTS. The rest of this playbook assumes that you have all dependency binaries in your system environment path variable. At a high level, the steps are -
+
+.. code-block:: text
+
+    1. Start ElasticSearch
+    2. Start Kibana
+    3. Configure Workspace listeners to write events to Search mongo db
+    4. Restart Workspace service
+    5. Create a Workspace data type
+    6. Configure KBaseSearchEngine
+    7. Start worker
+    8. Start coordinator
+    9. Verify ElasticSearch index
+
+1. Open a new terminal and start ElasticSearch.
+
+.. note::
+
+    Elastic Search can only be started up by a non-root user
+
+.. code-block:: bash
+
+    $ elasticsearch
+
+2. Open a new terminal and start Kibana.
+
+.. code-block:: bash
+
+   $ kibana
+
+3. Configure the Workspace listeners to write events to the Search mongodb.
+
+.. code-block:: bash
+
+    $ gedit [PATH_TO_YOUR_WORKSPACE_DIR]/deploy.cfg
+
+Make the following changes -
```

Comment:
Would "Add the following lines" make more sense here? It sounds like the user should be modifying exiting lines otherwise. @tiramisu24 was tripped up by this briefly.

Reply:
Fixed.